charts/kubescape-operator/templates/node-agent/default-rule-binding-namespaced.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and .Values.alertCRD.installDefault .Values.alertCRD.scopeNamespaced }}
+apiVersion: kubescape.io/v1
+kind: RuntimeRuleAlertBinding
+metadata:
+  name: all-rules-default-namespace
+  namespace: default
```

Comment:
chart NS, not default

Reply:
Why? This means you will have the alerts only on the KS namespace. (the RB crd affects only the namespace and not other namespaces, users that do not have cluster-wide permissions will need to create similar CRDs on the namespaces they want to create alerts)

Reply:
In any case we have the namespace selector in the object.
If the RB is rab is namespaced scope is just a location of the object, but has no effect on the logic.

Reply:
I reverted this behavior for now, but we will need to think of a better solution